addon/src/mocks/mock-request.js

```diff
@@ -1,18 +1,12 @@
 import { assert } from '@ember/debug';
-import { typeOf } from '@ember/utils';
-import {
-  isEmptyObject,
-  isEquivalent,
-  isPartOf,
-  param,
-  toParams,
-} from '../utils/helper-functions';
```

Comment:
aimed to reduce the need for bespoke code / utility functions, makes it a lot harder for new maintainers to know exactly what each function is supposed to do, without spending lots of time learning them